restapi/src/main/java/io/stargate/web/resources/Db.java

```diff
@@ -33,8 +33,6 @@
 import java.util.stream.Collectors;
 
 public class Db {
-
-  private final DataStore dataStore;
```

Comment:
`DataStore` was no longer used for anything before this PR: factory used when creating new instances. So just removing as unused.